### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by elsygonzalez**
Learning how to create a pull request and use github.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+
+My name is Elsy Gonzalez, I am a third year Ph.D student in **Political Science** with a specialization in *International Relations*.
+
+I have a BA degree in Political Science and International Relations from the *Centro de Investigación y Docencia Economicas* in Mexico City, Mexico.
+
+## Interests
+
+My substantive interests are the following:
+
+1. International recognition of statehood
```
Comment (bensoltoff) [inline README.md:14]: With enumerated lists, you don't need to manually specify the value. Markdown will render it based on the number of previous items. So this list could be written

```
1. International recognition of statehood
1. Recognition of secessionist and irredentist movements
1. Diaspora lobbying
    a. I am specifically interested in diaspora lobbying efforts for the purpose of garnering recognition for the independence of a secessionist/irredentist movement.
```

1. International recognition of statehood
1. Recognition of secessionist and irredentist movements
1. Diaspora lobbying
    a. I am specifically interested in diaspora lobbying efforts for the purpose of garnering recognition for the independence of a secessionist/irredentist movement.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+
+2. Recognition of secessionist and irredentist movements
+
+3. Diaspora lobbying
+
+    a. I am specifically interested in diaspora lobbying efforts for the purpose of garnering recognition for the independence of a secessionist/irredentist movement.
+    
+For more information, please see my description on the **UChicago** *Political Science* website at https://political-science.uchicago.edu/directory/elsy-gonzalez
```
Comment (bensoltoff) [inline README.md:22]: To explicitly add a URL but have the link actually be meaningful text, use the syntax `[]()`.

```
For more information, please see my description on the [**UChicago** *Political Science* website](https://political-science.uchicago.edu/directory/elsy-gonzalez)
```

For more information, please see my description on the [**UChicago** *Political Science* website](https://political-science.uchicago.edu/directory/elsy-gonzalez)

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |       x    |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Lots of Markdown elements demoed
* No extraneous files in the repo

**Comment by ghost** [comment]
Evaluation HW-xx
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |   x        |              |            |

Remarks:
Good use of lists
Nice incorporation of a link, next time try embedding it within text
Clear and concise language throughout

**Pull request closed without merging**